donations/views.py

```diff
@@ -111,22 +119,26 @@ def donation_complete_paypal(request):
     This view listens to a notification made from paypal when someone makes
     a donation, it validates the data and then stores the donation.
     """
-    params = request.POST.copy()
-    params.update({'cmd': '_notify-validate'})
```

Comment:
Removing this line makes the tests fail, for some reason I did not see the tests were failing before merging the PR, but I saw before after running the tests locally. Fixed this in https://github.com/MTG/freesound/commit/6271a334995953e2e8f97b97539a4ba9a0a864c0